test.js

```diff
@@ -22,3 +22,10 @@ test('remote', function (t) {
     t.ok(bytes < 500)
   })
 })
+
+test('uglify error should not crash the application', function (t) {
+  t.doesNotThrow(function () {
+    size('qs', {version: '6.0.1'}, function () {})
```

Comment:
you need to manage the async operation in this test. you can't just end it immediately.